website/src/releases/2020-08/apollo-3.0.mdx

```diff
@@ -0,0 +1,83 @@
+---
+date: 2020-08-25
```

Comment:
Just a guess, we might need to change this.

Reply:
Yes, let's postpone this to September.